### PR TITLE
Fix (and test!) graphql endpoint lens_scope. Add a smoke test to e2e tests.

### PR DESCRIPTION
**Pull request by wimax-grapl**
### Which issue does this PR correspond to?
None. Andrea reported a bug.

### What changes does this PR make to Grapl? Why?
- I discovered a bug where I was setting stuff to `dgraph_type: undefined` and then we were checking *if that key existed* not its value
- added a smoke test to prevent this in the future

### How were these changes tested?
added tests

### CODE REVIEW

File: src/js/engagement_view/src/components/engagementView/sidebar/tables/toggleLensTable.tsx [hunk 1]
```diff
@@ -62,7 +62,7 @@ export function ToggleLensTable({ setLens }: ToggleLensTableProps) {
 					}
 				}
 			);
-		}, 1000);
+		}, 5000);
```
Comment (wimax-grapl) [inline src/js/engagement_view/src/components/engagementView/sidebar/tables/toggleLensTable.tsx:65]: this is an annoyance I've wanted to change for a while - the constant pinging of the graphql endpoint is slowed down a bit.

File: src/js/graphql_endpoint/server.ts [hunk 2]
```diff
@@ -77,6 +86,7 @@ app.use(
     return {
       schema: getRootQuerySchema(),
       graphiql: IS_LOCAL,
+      customFormatErrorFn: IS_LOCAL ? customFormatErrorFnForDebugging : undefined,
```
Comment (wimax-grapl) [inline src/js/graphql_endpoint/server.ts:89]: Gives you a bit more information in errors returned over graphql

File: src/python/graphql_endpoint_tests/tests.py [hunk 3]
```diff
@@ -72,55 +72,3 @@ def _query_graphql_endpoint_for_lenses(
     """
     resp = gql_client.query(query)
     return resp["lenses"]
-
-
-def _query_graphql_endpoint_for_scope(
```
Comment (wimax-grapl) [inline src/python/graphql_endpoint_tests/tests.py:77]: moved to the graphql_client

File: src/js/graphql_endpoint/server.ts [hunk 4]
```diff
@@ -61,6 +62,14 @@ const corsDelegate = (req: cors.CorsRequest, callback: CorsCallback): void => {
   callback(null, corsOptions); // callback expects two parameters: error and options
 };
 
+function customFormatErrorFnForDebugging(error: GraphQLError) {
+  return {
```
Comment (andrea-grapl) [inline src/js/graphql_endpoint/server.ts:66]: Excellent!

**Review by andrea-grapl — APPROVED** [review]

**Pull request merged**